image.go

```diff
@@ -20,6 +20,8 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
+	// Image provides access to internal image info
```

Comment:
@ijc There are 2 reasons:
1) Containerd image store is indexed by image reference (docker.io/library/busybox:latest), at least the client and `ctr` does so; however, in cri-containerd we need to index image by image ID and maintain the ID->Other References mapping.
2) Containerd put image into store before unpacking the image, it means that an image in image store may not be a "ready-to-use" image. So we need to maintain our own list of the "ready-to-use" image. Of course we could always check whether corresponding snapshots are there before returning information about the image, but I feel like it would be easier to maintain our own list now.